cyclops/evaluate/metrics/precision_recall_curve.py

```diff
@@ -328,108 +332,113 @@ def compute(self) -> Tuple[np.ndarray, np.ndarray, np.ndarray]:
-                [1.        , 0.66666667, 0.        , 0.        ]]),
-        array([0. , 0.5, 1. ]))
+    task : Literal["binary", "multiclass", "multilabel"]
+        The task for which the precision-recall curve is computed.
+    thresholds : int or list of floats or numpy.ndarray of floats, default=None
+        Thresholds used for computing the precision and recall scores. If int,
+        then the number of thresholds to use. If list or array, then the
+        thresholds to use. If None, then the thresholds are automatically
+        determined by the sunique values in ``preds``
+    pos_label : int, default=1
+        Label to consider as positive for binary classification tasks.
+    num_classes : int, optional
+        The number of classes in the dataset. Required if ``task`` is
+        ``"multiclass"``.
+    num_labels : int, optional
+        The number of labels in the dataset. Required if ``task`` is
+        ``"multilabel"``.
 
+    Examples
+    --------
+    (binary)
+    >>> from cyclops.evaluation.metrics import PrecisionRecallCurve
+    >>> target = [1, 1, 1, 0]
+    >>> preds = [0.6, 0.2, 0.3, 0.8]
+    >>> metric = PrecisionRecallCurve(task="binary", thresholds=None)
+    >>> metric(target, preds)
+    (array([0.75      , 0.66666667, 0.5       , 0.        , 1.        ]),
+    array([1.        , 0.66666667, 0.33333333, 0.        , 0.        ]),
+    array([0.2, 0.3, 0.6, 0.8]))
+    >>> metric.reset_state()
+    >>> target = [[1, 0, 1, 1], [0, 0, 0, 1]]
+    >>> preds = [[0.5, 0.4, 0.1, 0.3], [0.9, 0.6, 0.45, 0.8]]
+    >>> for t, p in zip(target, preds):
+    ...     metric.update_state(t, p)
+    ...
+    >>> metric.compute()
+    (array([0.5       , 0.42857143, 0.33333333, 0.4       , 0.5       ,
+        0.33333333, 0.5       , 0.        , 1.        ]),
+    array([1.  , 0.75, 0.5 , 0.5 , 0.5 , 0.25, 0.25, 0.  , 0.  ]),
+    array([0.1 , 0.3 , 0.4 , 0.45, 0.5 , 0.6 , 0.8 , 0.9 ]))
 
-    Examples (multilabel)
-    ---------------------
-        >>> from cyclops.evaluation.metrics import PrecisionRecallCurve
-        >>> target = [[0, 1], [1, 0]]
-        >>> preds = [[0.1, 0.9], [0.8, 0.2]]
-        >>> metric = PrecisionRecallCurve(task="multilabel", num_labels=2,
-        ...     thresholds=3)
-        >>> metric(target, preds)
-        (array([[0.5, 1. , 0. , 1. ],
-                [0.5, 1. , 0. , 1. ]]),
-        array([[1., 1., 0., 0.],
-                [1., 1., 0., 0.]]),
-        array([0. , 0.5, 1. ]))
-        >>> metric.reset_state()
-        >>> target = [[[0, 1], [1, 0]], [[1, 0], [0, 1]]]
-        >>> preds = [[[0.1, 0.9], [0.8, 0.2]],
-        ...         [[0.1, 0.9], [0.8, 0.2]]]
-        >>> for t, p in zip(target, preds):
-        ...     metric.update_state(t, p)
-        >>> metric.compute()
-        (array([[0.5, 0.5, 0. , 1. ],
-                [0.5, 0.5, 0. , 1. ]]),
-        array([[1. , 0.5, 0. , 0. ],
-                [1. , 0.5, 0. , 0. ]]),
-        array([0. , 0.5, 1. ]))
+    Examples
```

Comment:
I think you can only have one `Examples` section, but then use (binary), (multiclass) and line spacing to separate those. Right now, with multiple `Examples` mentioned, sphinx gets confused and doesn't render them.

Reply:
There are still some errors about indentation, but I don't know how to fix those.

Reply:
That's fine, can fix those later!

Reply:
Do you know which indentation it's complaining about though?

Reply:
Hmm I recall fixing some of those a while back. These don't seem to affect the docs rendering though. I'll investigate them in a bit. Have added to my TODOs.